dhdt/generic/handler_sentinel2.py

```diff
@@ -250,6 +250,31 @@ def get_geom_for_tile_code(tile_code, geom_dir=None, geom_name=None):
         print('MGRS tile code does not seem to exist')
     return geom.ExportToWkt()
 
+def get_tile_codes_from_geom(geom, geom_dir=None, geom_name=None):
```

Comment:
This is a placeholder, should really add this and make it uniform to `get_geom_for_tile_code` and `get_bbox_from_tile_code` in #61